tests/onnxruntime/test_modeling.py

```diff
@@ -1242,7 +1390,20 @@ def test_compare_to_io_binding(self, model_id):
 
 
 class ORTModelForCausalLMIntegrationTest(unittest.TestCase):
-    SUPPORTED_ARCHITECTURES = ("gpt2",)
+    SUPPORTED_ARCHITECTURES = (
+        "gpt2",
+        "bloom",
+        "codegen",
+        "bigbird_pegasus",
+        "bart",
```

Comment:
Why you put bart, mbart and marian here? They are seq2seq models.

Reply:
This was just copy paste, basically because of 

![image](https://user-images.githubusercontent.com/9808326/210757290-c07152de-edb8-4dd2-8700-c128de244199.png)

As I understand reading `modeling_mbart.py`, there is a `MBartForCausalLM` that does not use the encoder, and a `MBartForConditionalGeneration` that does. I guess that's because of it?